ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -74,6 +75,50 @@
 
 public class SampleData {
 
+    private static Supplier<CCDCase.CCDCaseBuilder> getCCDBuilderWithDefault = () -> CCDCase.builder()
```

Comment:
Done

Reply:
Minor Suggestion: may be rename the variable 'getCCDBuilderWithDefault' to 'ccdBuilderWithDefault' because the supplier has get() by default